src/pages/Donate/DonateSupportUs.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+	<www-page
+		id="support-us-landing"
+	>
+		<div class="row page-content">
+			<div class="small-12 large-7 columns">
+				<h1 class="donate-headline" v-html="headlineCopy"></h1>
+				<div class="donate-subhead" v-html="subHeadCopy"></div>
+				<donate-form
+					:key="1"
+					:button-text="buttonText"
+					:data="donationValues"
+					:form-disclaimer="formDisclaimer"
+					:form-submit-analytics="formSubmitAnalytics"
+					:show-disclaimer="false"
+				/>
+			</div>
+			<div class="small-12 large-5 columns">
+				<h4>Loans vs. Donations</h4>
+				<p>Kiva does not take a cut of loans, optional donations fund our operations.</p>
+			</div>
+			<div class="small-12 columns">
+				<h4>Frequently asked questions</h4>
+			</div>
+		</div>
+	</www-page>
+</template>
+
+<script>
+import gql from 'graphql-tag';
+
+import WwwPage from '@/components/WwwFrame/WwwPage';
+import { processPageContentFlat } from '@/util/contentfulUtils';
+import { documentToHtmlString } from '~/@contentful/rich-text-html-renderer';
+import DonateForm from './DonateForm';
+
+const pageQuery = gql`query donateContent {
+	contentful {
+		entries (contentType: "page", contentKey: "web-donate-support-us")
```

Comment:
for almost all the other pages we're mapping these keys to the URL, any reason why this isn't `donate/supportus`  or `supportus` ?

Reply:
Over this is using a derivative of the url with a prefix for the intended platform. Generally, I don't think we should be using `/` in our keys or string identifiers.

This also aligns with the system we are using for mobile to prefix content for the apps.

Reply:
Ok sounds good, I wasn't aware we were using prefixes like that

Reply:
I'll admit the "app-" prefix is new as well but certainly a welcome addition now that Mobile is moving fully into setting up their own pages with their own content entries. You'll notice in contentful that ALL of the entries for this page are prefixed in a similar manner which is likely the proper paradigm to start following across all new content.